src/elmer_compiler.erl

```diff
@@ -60,6 +60,10 @@ dump(erlsrc, Filename, Erl) ->
     Bin = erlang:iolist_to_binary(Src),
     {Filename, Bin};
 
+dump(binary, Filename, Erl) ->
```

Comment:
Actually, the original idea was to have a `dump({beam, OutputDirectory}, Filename, Erl)` function, that's what would get called by the [rebar3_elmer plugin](https://github.com/elmer-compiler/rebar3_elmer/blob/master/src/rebar3_elmer_compile.erl#L58)